src/test/a11y/mocks.ts

```diff
@@ -75,6 +75,9 @@ mock('response/guards/guardFactory', {
   GuardFactory: {
     create: () => {
       return justForwardRequestHandler.requestHandler
+    },
+    createAsync: () => {
+      return justForwardRequestHandler.requestHandler
```

Comment:
Shouldn't that return an `async` function?

Reply:
It worked fine because both methods return just request handler which either evaluates in sync or async way and framework is able to handle both. Anyway I changed it to be more explicit:

```
    create: () => {
      return (req: express.Request, res: express.Response, next: express.NextFunction): void => {
        next()
      }
    },
    createAsync: () => {
      return async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
        next()
      }
    }
```